service/common/src/testFixtures/java/org/apache/polaris/service/TestServices.java

```diff
@@ -147,7 +149,6 @@ public Builder stsClient(StsClient stsClient) {
 
     public TestServices build() {
       PolarisConfigurationStore configurationStore = new MockedConfigurationStore(config);
-      PolarisDiagnostics polarisDiagnostics = Mockito.mock(PolarisDiagnostics.class);
```

Comment:
wondering:
should we annotate `PolarisDiagnostics` with https://errorprone.info/api/latest/com/google/errorprone/annotations/DoNotMock.html ?

currently there are no legitimate cases for mocking it in the codebase it seems (but there could be in the future)